add `OpInfo` for `torch.nn.functional.pairwise_distance`

pmeier opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #65681
* #65680
* #65469
* #65468
* #65467
* __->__ #65460



cc @albanD @mruberry @jbschlosser @walterddr

Differential Revision: [D31111701](https://our.internmc.facebook.com/intern/diff/D31111701)

pmeier committed:
add `OpInfo` for `torch.nn.functional.pairwise_distance`

[ghstack-poisoned]

pmeier committed:
Update on "add `OpInfo` for `torch.nn.functional.pairwise_distance`"




cc albanD mruberry jbschlosser walterddr

[ghstack-poisoned]

zou3519 commented:
@zou3519 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31111701).

pmeier committed:
Update on "add `OpInfo` for `torch.nn.functional.pairwise_distance`"




cc albanD mruberry jbschlosser walterddr

Differential Revision: [D31111701](https://our.internmc.facebook.com/intern/diff/D31111701)

[ghstack-poisoned]

pmeier committed:
Update on "add `OpInfo` for `torch.nn.functional.pairwise_distance`"




cc albanD mruberry jbschlosser walterddr

Differential Revision: [D31111701](https://our.internmc.facebook.com/intern/diff/D31111701)

[ghstack-poisoned]

pmeier commented on torch/testing/_internal/common_methods_invocations.py:
```diff
@@ -5978,6 +5997,9 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
         numel = torch.tensor(t.shape)[kwargs.get('dim')].prod()
         yield ((), {'correction': numel // 2})
 
+def ref_pairwise_distance(input1, input2):
+    pass
+
```
Oops, that shouldn't be here. I've already removed locally and will push this with the next stack update.

zou3519 commented:
@zou3519 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31111701).

pmeier committed:
Update on "add `OpInfo` for `torch.nn.functional.pairwise_distance`"




cc albanD mruberry jbschlosser walterddr

Differential Revision: [D31111701](https://our.internmc.facebook.com/intern/diff/D31111701)

[ghstack-poisoned]

pmeier committed:
Update on "add `OpInfo` for `torch.nn.functional.pairwise_distance`"




cc albanD mruberry jbschlosser walterddr

Differential Revision: [D31111701](https://our.internmc.facebook.com/intern/diff/D31111701)

[ghstack-poisoned]

pmeier committed:
Update on "add `OpInfo` for `torch.nn.functional.pairwise_distance`"




cc albanD mruberry jbschlosser walterddr

Differential Revision: [D31111701](https://our.internmc.facebook.com/intern/diff/D31111701)

[ghstack-poisoned]
